WordPress/wordcamp.org
# WordCamp Central: Add global sponsors "block" to homepage

Swaps out the Latest Tweets column on the WordCamp Central [homepage](https://central.wordcamp.org) for a randomized list of three Gold-level sponsors. The seed for the randomize function changes every hour.

The code that outputs the sponsor list is structured similarly to WordCamp's Sponsor block, so that it would be easier to convert this into an actual block in the future.

Fixes #306

**Screenshot (Updated):**

![widget-logos](https://user-images.githubusercontent.com/916023/72482205-c4e06200-37b1-11ea-9ef1-e3a88652bd48.jpg)

~Right now we are lucky that all of the Gold-level sponsors have logos with similar aspect ratios. However, I think further design thinking is needed here about how best to accommodate one or more logos that aren't necessarily all in landscape orientation. @karmatosed~

~Also, does the heading make sense, given the link to the rest of the sponsors at the bottom? Should that link be there? @andreamiddleton~

**To test (Updated):**

* Load the WordCamp Central homepage in your sandbox.
* The widgets should already be there, I think, because I added them via my sandbox.
* Ensure that the third column below the hero contains Gold-level sponsors' logos, plus a link to the Global Sponsorship page.
* Try reloading the page a couple of times and make sure the order of the logos changes.

**Pre-deploy**

* [ ] Receive and confirm square logo assets from each of the Gold-level global sponsors. Update the widgets accordingly.
